src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -203,33 +202,20 @@ internal static TypedData ToRpcHttp(this HttpRequest request, ILogger logger, Ca
                             body = rawBodyString;
                         }
                     }
-                    else if (string.Equals(mediaType.MediaType, "application/octet-stream", StringComparison.OrdinalIgnoreCase) ||
-                        mediaType.MediaType.IndexOf("multipart/", StringComparison.OrdinalIgnoreCase) >= 0)
+                    else if (IsMediaTypeOctetOrMultipart(mediaType))
                     {
                         body = bytes;
-                        if (!IsRawBodyBytesRequested(capabilities))
-                        {
-                            rawBodyString = Encoding.UTF8.GetString(bytes);
-                        }
                     }
-                }
+               }
                 // default if content-tye not found or recognized
-                if (body == null && rawBodyString == null)
-                {
+               if (body == null)
+               {
                     var reader = new StreamReader(request.Body, Encoding.UTF8);
-                    body = rawBodyString = reader.ReadToEnd();
-                }
+                    body = reader.ReadToEnd();
+               }
 
-                request.Body.Position = 0;
-                http.Body = body.ToRpc(logger, capabilities);
-                if (IsRawBodyBytesRequested(capabilities))
-                {
-                    http.RawBody = bytes.ToRpc(logger, capabilities);
-                }
-                else
-                {
-                    http.RawBody = rawBodyString.ToRpc(logger, capabilities);
-                }
+               request.Body.Position = 0;
+               http.Body = body.ToRpc(logger, capabilities);
```

Comment:
@maiqbal11 - Can you confirm if python worker send IsRawBodyBytesRequested by default?